### PR TITLE
Manage the management environment with CLI

### PR DESCRIPTION
**Pull request by iverberk**
This PR brings the management environment under control of the CLI. It extends the documentation with all the necessary steps to fully setup the management environment until it is ready for CI/CD.

### CODE REVIEW

File: management/vault/config/terraform.tf [hunk 1]
```diff
@@ -29,7 +29,7 @@ terraform {
   backend "azurerm" {
     resource_group_name = "terraform-state"
     container_name      = "management"
-    key                 = "vault-config.tfstate"
+    key                 = "azure/management/management-vault-config.tfstate"
```
Comment (arminc) [inline management/vault/config/terraform.tf:32]: The keys are already stored in "management" container, management is always in azure. I think it is redundant to say arzure/management/management-X.

File: docs/opsteady/management-setup.md [hunk 2]
```diff
@@ -116,21 +117,100 @@ The certificate authority file for Vault can be downloaded from `https://${manag
...
-terraform apply -var-file=../../defaults/vault-config.tfvars -var management_vault_config_token=$VAULT_ROOT_TOKEN_FROM_VAULT_INFRA_RUN
+# This command will give you some warnings about values for undeclared variables. This is expected and can be ignored.
+terraform apply -compact-warnings -var-file=../../defaults/infra.tfvars.json -var-file=../../defaults/vault-config.tfvars.json -var management_vault_config_token=$VAULT_ROOT_TOKEN_FROM_VAULT_INFRA_RUN
 ```
 
+NOTE: If you did not add yourself to the "platform-admin" group while creating the management infra then you need to add yourself now via the Azure Portal.
+
 You should now be able to login via OIDC on `https://vault.management.${management_infra_domain}` and see the configurations.
+
+## 05 Revoke the Vault root token
```
Comment (arminc) [inline docs/opsteady/management-setup.md:138]: Let's add a comment to security-compliance that we are doing this.

File: management/vault/config/terraform.tf (hunk 1 above)
Comment (iverberk) [reply management/vault/config/terraform.tf:32]: Discussed that we'll keep it like this to have consistency in the CLI between management and platform. Management is a special case of an Azure platform.

File: docs/opsteady/management-setup.md (hunk 2 above)
Comment (iverberk) [reply docs/opsteady/management-setup.md:138]: This will be added when we write the ADR for Vault.

**Pull request merged**